docs/notifications.md

```diff
@@ -0,0 +1,250 @@
+
+By leveraging the Notifications feature, you can automate your licensing workflows and streamline your business processes. You can use the notifications to trigger actions like sending automated emails to customers, updating internal databases or CRM systems, or generating reports based on events occurring within your licensing environment.
+
+Setup
+-----
+
+Manage your NetLicensing Notifications from the *"Settings / Notifications"* section. Here, you can view, deactivate, delete, and create notifications.
+
+### Creating a New Notification
+
+1. Click on "Add Notification" to start the creation process
+2. In the "Create Notification" view, fill in the required attributes
+3. Click "Create" to create notification
+
+<a href="assets/images/notifications.png" class="imagelink" data-lightbox="notifications" data-title="Notifications" data-alt="Notifications">
+    <img src="assets/images/notifications.png" />
+</a>
+
+### Notification Attributes
+
+- **Name**: The name of the notification.
+- **Number**: A unique notification number. If left empty, a number will be generated automatically.
+- **Events**: One or more events that will trigger the notification. Options include:
+  - **Create Customer**: Triggered when a new customer is created.
+  - **Create License**: Triggered when a new license is created.
+  - **Change Warning Level**: Triggered on customer's [warning level](warning-level) changes (e.g., from `GREEN` to `YELLOW` or `YELLOW` to `RED`).
+- **Type**: The type of notification. Currently, only `WEBHOOK` is supported.
+- **Endpoint**: The URL where an HTTP POST request with the specified payload will be sent. **Note:** The endpoint must use the secured HTTPS protocol with a public SSL/TLS certificate.
+- **Payload**: The specific payload for the event can be defined here.
+
+Payload Structure
+-----------------
+
+Notification payload can be configured to include specific data related to the event that triggered the notification. The payload is sent as a JSON object in the body of the HTTP POST request to the specified endpoint URL. The structure of the payload depends on the event type and the data associated with the event.
+
+The standard payload structure includes the following fields:
+
+- **Event**: `${event}` - full event object
+- **Data**: `${event.data}` - event-specific data
+- **Name**: `${event.name}` - event name; can be one of the following: `LICENSEE_CREATED`, `LICENSE_CREATED`, `WARNING_LEVEL_CHANGED`
+- **Timestamp**: `${event.timestamp}` - timestamp of the event
+
+Below are examples of the payload structure (`event.data` element) for different events:
+
+<div>LICENSEE_CREATED</div>
+{: .code-example .ml-5 .code-header }
+```json
+{
+    ...
+    "items": {
+        "item": [
+            {
+                "property": [
+                    {
+                        "value": "IADGNSGZG",
+                        "name": "number"
+                    },
```

Comment:
Is this notification format up-to-date?

Reply:
Yes, this is the current released format from PROD.

Changes to the format will be addressed with this [PR #204](https://github.com/Labs64/NetLicensing-Core/pull/204). Therefore, we may need to revisit the samples after the PR is released.